app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt

```diff
@@ -321,6 +323,8 @@ class MainActivity : AppCompatActivity(), ColorPickerDialogListener {
          */
         val reloadLibraryEvent = Event<Boolean>()
 
+        /** For battery optimisation **/
+        private var isAppOptimised : Boolean by UserPreferenceDelegate("isAppOptimised", false)
```

Comment:
UserPreferenceDelegate is just a wrapper for set/get key, so restoring a backup skips this even if it is not optmized